src/js/components/Search.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Search = (/* props */) => (
```

Comment:
Resolved. Now a PureComponent, instead of stateless. MUI field added. Export updated.